test/Microsoft.AspNet.Mvc.ViewFeatures.Test/Rendering/HtmlHelperHiddenTest.cs

```diff
@@ -835,6 +835,40 @@ public void HiddenFor_DoesNotUseAttributeValue()
             Assert.Equal(expected, HtmlContentUtilities.HtmlContentToString(result));
         }
 
+        [Fact]
+        public void HiddenHelpers_UsesSpecifiedExpression()
+        {
+            // Arrange
+            var helper = DefaultTemplatesUtilities.GetHtmlHelper();
+
+            // Act
+            var hiddenResult = helper.Hidden("Property1");
+            var hiddenForResult = helper.HiddenFor(m => m.Property1);
```

Comment:
Not a fan of doing two independent tests in the same `[Fact]` though we have some ugly examples in these HTML helper tests.

Split into separate methods. Or (perhaps) pass `Func<IHtmlHelper<ForgetTheType>, IHtmlContent>` into a `[Theory]` (since the content matches here).